modular_zubbers/maps/offstation/persistence/area.dm

```diff
@@ -6,6 +6,7 @@
 	icon_state = "syndie-ship"
 	outdoors = FALSE
 	ignore_weather_sfx = TRUE
+	flags_1 = null // Stops SSDecay from affecting the Persistance
```

Comment:
```suggestion
	flags_1 = parent_type::flags_1 & ~(CAN_BE_DIRTY_1) // Stops SSDecay from affecting the Persistance
```
if upstream ever adds anything else to area flags this way we don't just null it too and end up with mystery bugs